pyccel/parser/semantic.py

```diff
@@ -1037,40 +1062,44 @@ def _ensure_target(self, rhs, d_lhs):
             d_lhs['memory_handling'] = 'alias'
             rhs.base.is_target = not rhs.base.is_alias
 
-    def _assign_lhs_variable(self, lhs, d_var, rhs, new_expressions, is_augassign,arr_in_multirets=False, **settings):
+    def _assign_lhs_variable(self, lhs, d_var, rhs, new_expressions, is_augassign,arr_in_multirets=False):
         """
-        Create a lhs based on the information in d_var
-        If the lhs already exists then check that it has the expected properties.
+        Create a variable from the left-hand side (lhs) of an assignment.
+        
+        Create a lhs based on the information in d_var, if the lhs already exists
+        then check that it has the expected properties.
 
         Parameters
         ----------
         lhs : PyccelSymbol (or DottedName of PyccelSymbols)
-            The representation of the lhs provided by the SyntacticParser
+            The representation of the lhs provided by the SyntacticParser.
 
         d_var : dict
-            Dictionary of expected lhs properties
+            Dictionary of expected lhs properties.
 
         rhs : Variable / expression
             The representation of the rhs provided by the SemanticParser.
             This is necessary in order to set the rhs 'is_target' property
-            if necessary
+            if necessary.
 
-        new_expression : list
+        new_expressions : list
             A list which allows collection of any additional expressions
-            resulting from this operation (e.g. Allocation)
+            resulting from this operation (e.g. Allocation).
 
         is_augassign : bool
             Indicates whether this is an assign ( = ) or an augassign ( += / -= / etc )
             This is necessary as the restrictions on the dtype are less strict in this
-            case
+            case.
 
         arr_in_multirets : bool
             If True, rhs has an array in its results, otherwise, it should be set to False.
             It helps when we don't need lhs to be a pointer in case of a returned array in
             a tuple of results.
 
-        settings : dictionary
-            Provided to all _visit_ClassName functions
+        Returns
+        -------
+        pyccel.ast.variable.Variable
+            The representation of the lhs provided by the SyntacticParser.
```

Comment:
There is an error here:
```suggestion
            The representation of the lhs provided by the SemanticParser.
```